test/Grpc.IntegrationTests/OrchestrationErrorHandling.cs

```diff
@@ -543,8 +545,8 @@ static void ValidateInnermostFailureDetailsChain(TaskFailureDetails? failureDeta
                     {
                         // Outer failure represents the orchestration failure
                         Assert.NotNull(ex.FailureDetails);
-                        Assert.True(ex.FailureDetails.IsCausedBy<TaskFailedException>());
-                        Assert.Contains("ThrowException", ex.FailureDetails.ErrorMessage);
```

Comment:
This change worries me - our tests were checking that the activity name was present in the exception message, which I think is important. Even if this change means that this information is in some InnerFailure, can we still make sure that this information is present somewhere in the exception details?